spec/frosting/repository_spec.rb

```diff
@@ -43,12 +56,31 @@ class Test::OtherResource ; end
     end
 
     describe ".present_collection" do
-      it "presents each item in the collection with options" do
-        resource = double
-        described_class.should_receive(:present)
+      class Collection < Array
+        def test_method
+          "cats"
+        end
+      end
+
+      let(:resource) { double }
+      let(:presented_resource) { double }
+      let(:presented_collection) do
+        described_class.present_collection(Collection.new.push(resource), {option: :val})
+      end
+
+      before do
+        described_class.stub(:present)
```

Comment:
It seems that only the first example here cares about this stub setup, so I might put it right in there and remove the `before`. That would help clarify the bit in the example's description about caring about options, as well.